.github/ISSUE_TEMPLATE/transfer-project-in.yml

```diff
@@ -41,13 +41,35 @@ body:
   - type: checkboxes
     id: options
     attributes:
-      label: Which of the following options would you like?
+      label: Do you use test PyPI or want to test the packaging process?
       description: "These are entirely up to you and can be changed at a later time."
       options:
         - label: Does the project have a test.pypi.org project?
         - label: Would you like to test your deployments with uploads to test.pypi.org (you'll need to create the project first)?
-        - label: Would you like to use GitHub's merge commits?
-        - label: Would you like to use GitHub's rebase and merge?
-        - label: Would you like to use GitHub's squash and merge?
-        - label: Would you like to use GitHub's Discussions?
-        - label: Would you like to use GitHub's Wiki?
+  - type: markdown
+    attributes:
+      value: |
+        Please copy and paste the following code block and edit it to meet your needs for the repository.
+
+        ```
+          "your-repo-here" = {
+            description          = "Your description here"
+            allow_merge_commit = true
+            allow_rebase_merge = true
+            allow_squash_merge = true
+            allow_update_branch = true
```

Comment:
What do these options mean? Can we link to a reference in the GitHub docs?

Reply:
That's a good idea

Reply:
- Merge methods: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github
- Configuring merge commits: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-merging-for-pull-requests
- Configuring squash merge: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests
- Configuring rebase merge: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-rebasing-for-pull-requests
- Configuring ability to allow branch updating: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-suggestions-to-update-pull-request-branches
- Configuring auto-merge: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-auto-merge-for-pull-requests-in-your-repository

I'm not sure those URLs could be any longer.

Also we should expose the auto merge option from the default list.